### PR TITLE
updated description in readme.md

### PR DESCRIPTION
**Pull request by masonse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-Warning: This relies on an unreleased version of Dispatch.
-
 # Dispatch Catalog
 
-The official [Dispatch](https://docs.d2iq.com/ksphere/dispatch/latest/) catalog. Currently, this only hosts our Starlark standard library, however, this will grow to include other resources.
+![Image description](./images/dispatch-service-icon--medium.png)
+
+**Accelerate your CI/CD**
+
+D2iQ's [Dispatch](https://d2iq.com/solutions/ksphere/dispatch) helps organizations deliver CI/CD pipelines that support faster code development and deployment. The Dispatch catalog is a repository of libraries that Dispatch users can import into their pipelines.  Leverage the Dispatch catalog to accelerate common integration tasks such as testing, artifact management kubernetes resource usage and unit test execution. Bookmark or watch this page to leverage additional resources as they become available.
```
Comment (jbarrick-mesosphere) [inline README.md]: > Bookmark or watch this page to leverage additional resources as they become available.

If we link back to the git repository, we can recommend they use Github's "watch" feature to be notified on releases.

**Review by jbarrick-mesosphere — APPROVED** [review]

**Comment by jbarrick-mesosphere** [comment]
There's also docs/README.md which is the landing page for the Github pages site.

File: README.md (hunk 1 above)
Comment (jbarrick-mesosphere) [reply README.md]: Actually this is the doc for the repo and not the site so this makes sense as-is.

**Comment by masonse** [comment]
@jbarrick-mesosphere I updated the docs/readme.md as well.  Can you take a look? If all looks good, please merge.

File: docs/README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+![Image description](../images/dispatch-service-icon--medium.png)
+
+**Accelerate your CI/CD**
+
+[D2iQ's Dispatch](https://d2iq.com/solutions/ksphere/dispatch) helps organizations deliver CI/CD pipelines that support faster code development and deployment. The Dispatch catalog is a repository of libraries that Dispatch users can import into their pipelines.  Leverage the Dispatch catalog to accelerate common integration tasks such as testing, artifact management kubernetes resource usage and unit test execution. Watch the [repo](https://github.com/mesosphere/dispatch-catalog) to leverage additional resources as they become available.
+
+As the official [Dispatch](https://docs.d2iq.com/ksphere/dispatch/latest/) catalog, this currently hosts our Starlark standard libraries however, this will grow to include other resources and languages.
 
-Check out the [Starlark standard library documentation](./starlark/) for more information.
+Check out the [Starlark standard library documentation](./docs/starlark/) for more information.
```
Comment (jbarrick-mesosphere) [inline docs/README.md]:
```suggestion
Check out the [Starlark standard library documentation](./starlark/) for more information.
```

**Review by jbarrick-mesosphere — APPROVED** [review]

**Comment by jbarrick-mesosphere** [comment]
Thanks!

**Pull request merged**